### PR TITLE
Implemented: code to fetch the product store information for preparing order filter(#28ejub5)

### PR DESCRIPTION
**Pull request by ymaheshwari1**
### Related Issues

Closes #

### Screenshots of Visual Changes before/after (If There Are Any)

image

image

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [ ] I read and followed [contribution rules](https://github.com/hotwax/commerce-hub#contribution-guideline)

### CODE REVIEW

File: src/components/OrderFilters.vue [hunk 1]
```diff
@@ -121,7 +127,8 @@ export default defineComponent({
   computed: {
     ...mapGetters({
       query: 'order/getOrderQuery',
-      getShipmentMethodDesc: 'util/getShipmentMethod'
+      getShipmentMethodDesc: 'util/getShipmentMethod',
+      getEComStores: 'util/getEcomStores'
```
Comment (adityasharma7) [inline src/components/OrderFilters.vue]:
```suggestion
      eComStores: 'util/getEcomStores'
```

File: src/locales/en.json [hunk 2]
```diff
@@ -106,6 +106,7 @@
   "Product inventory detail": "Product inventory detail",
   "Product name": "Product name",
   "Product Store": "Product Store",
+  "Product Store Id": "Product Store Id",
```
Comment (adityasharma7) [inline src/locales/en.json]: I think we should show Product Store as title

File: src/views/FindOrder.vue [hunk 3]
```diff
@@ -194,7 +210,8 @@ export default defineComponent ({
       getProductStock: 'stock/getProductStock',
       isScrollable: 'order/isScrollable',
       query: 'order/getOrderQuery',
-      getShipmentMethodDesc: 'util/getShipmentMethod'
+      getShipmentMethodDesc: 'util/getShipmentMethod',
+      getEcomStores: 'util/getEcomStores'
```
Comment (adityasharma7) [inline src/views/FindOrder.vue]:
```suggestion
      ecomStores: 'util/getEcomStores'
```

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/views/FindOrder.vue (hunk 3 above)
Comment (ymaheshwari1) [reply src/views/FindOrder.vue]: Updated the getter name

File: src/locales/en.json (hunk 2 above)
Comment (ymaheshwari1) [reply src/locales/en.json]: Changed title to Product Store.

File: src/components/OrderFilters.vue (hunk 1 above)
Comment (ymaheshwari1) [reply src/components/OrderFilters.vue]: Done Sir

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**